src/components/questionnaire/questionnaire-table.tsx

```diff
@@ -25,34 +34,48 @@ import { usersAtom } from "src/atoms/user";
 import { userProfileAtom } from "src/atoms/auth";
 import type { User } from "src/generated/homeLambdasClient";
 
+interface EnhancedQuestionnaire extends Questionnaire {
+  tags?: string[];
+}
```

Comment:
Questionnaire as defined in the spec already includes the tags, so why do we need this enhancedQuestionnare interface- it can be removed and replaced with Questionnaire type